README.md

```diff
@@ -38,6 +38,20 @@ If your install of swift is located at `/swift` and you wish to install XCTest i
 ./build_script.py --swiftc="/swift/usr/bin/swiftc" --build-dir="/tmp/XCTest_build" --swift-build-dir="/swift/usr" --library-install-path="/swift/usr/lib/swift/linux" --module-install-path="/swift/usr/lib/swift/linux/x86_64"
 ```
 
+To run the tests, pass the `--test` option in combination with options to install XCTest in your active version of Swift:
+
+```sh
+./build_script.py --swiftc="/swift/usr/bin/swiftc" --build-dir="/tmp/XCTest_build" --swift-build-dir="/swift/usr" --library-install-path="/swift/usr/lib/swift/linux" --module-install-path="/swift/usr/lib/swift/linux/x86_64" --test
+```
+
+To add additional tests to XCTest:
+
+1. (Linux & OS X) Add a directory with the name of your test to the `Tests/Fixtures/Sources/` directory. For example, if your test is named "TwoFailingTestCases", make a directory named `Tests/Fixtures/Sources/TwoFailingTestCases`.
+1. (Linux & OS X) Add a `main.swift` file to the directory you added above. The file should contain comments, beginning with `// `. The test runner will verify that the output of `main.swift` matches your comments exactly. This is all you need to do to run tests on Linux--you may run the `build_script.py` example above to confirm the tests pass.
+1. (OS X) If you want your tests to run on OS X, open the `XCTest.xcodeproj` Xcode project and duplicate the `SingleFailingTestCase` target. Rename the target to match the name of your test. For example, if your test is named "TwoFailingTestCases", name the target `TwoFailingTestCases` as well.
```

Comment:
Yeah, no worries, thanks for all the feedback!! :100: 

> Right, but since you can build the executables on Linux, isn't it possible to build them on OS X without adding them to `*.xcodeproj`.

I actually would love to achieve all this without using Xcode projects, but I can't for the life of me figure out how to compile and link the SwiftXCTest.framework build products to the executables.

> (Just trying to understand this better, and perhaps wondering if those steps shouldn't be marked `Xcode` instead of `OS X`)

They could be. In practice, however, the two are the same--the SwiftXCTest.framework build system is tied to Xcode on OS X at the moment. As far as I can tell, it's not a Swift package, and is not meant to be built using llbuild--but correct me if I'm wrong here! :stuck_out_tongue: